### ISSUE TITLE
"Can't convert bad label back to class name"

### ISSUE THREAD

**Issue by robbiesaurus** [catboost/catboost#495]
Problem: {
Hi, I'm trying to use Catboost in Python for a multiclassification problem (the predicted value is a string from a list of possibilities). The model fits correctly but fails when attempting to predict with the error message "Can't convert bad label back to class name" . I don't understand what this error means or how to approach this problem, any help would be greatly appreciated. My code in question is below, the error is thrown while evaluating line 4:

```
def CBFitAndPredict(trainingTargets, trainingData, testingData, testingTargets):
    model = cb.CatBoostClassifier(loss_function='MultiClass')
    model.fit(X=trainingData, y=trainingTargets)
    predictions = model.predict(data=testingData)
```
}
catboost version: {0.10.2}
Operating System: {Windows 7}

**Comment by dbakshee** [comment]
Hi @robbiesaurus , could you provide a more complete example?
